recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "rubin-env" %}
 {% set version = "10.0.0" %}
-{% set build_number = 15 %}
+{% set build_number = 16 %}
```

Comment:
This is going to require an entirely new rubin-env version and so has to be handled separately. You can't require a new minimum version with an identical version number because all existing deployments would break.

Reply:
Should I change this to `version = "10.0.1"` and `build_number = 0` on this PR then or is this handled in some other way?

Reply:
A new rubin-env version needs planning and discussion with the build team. cc/ @mwittgen 
There are other tweaks that people have been asking for that are pending a new rubin-env release.

Reply:
Yes, you usually do that by filing an RFC with a DM implementation ticket, or at least the latter ticket, and then we can block the 11.0 release on that ticket.